libdevcore/Log.h

```diff
@@ -214,27 +214,6 @@ inline boost::log::formatting_ostream& operator<<(
     _strm.stream() << EthCyan "#" << _value.abridged() << EthReset;
     return _strm;
 }
-inline boost::log::formatting_ostream& operator<<(
```

Comment:
I remember that it was not working correctly for non-const variables without this overload https://github.com/ethereum/aleth/commit/d0af9c2cdc30d94084450481ffdc971afd74abfb

Reply:
It builds now.

Reply:
It was calling the wrong `operator<<` (generic `operator<<(formatting_ostream&, T&)` which ouput it differently from the const version)

Reply:
Ok, I only removed the h512 overloads.